examples/files.js

```diff
@@ -113,6 +113,7 @@ var files = {
 		"webgl_loader_prwm",
 		"webgl_loader_stl",
 		"webgl_loader_svg",
+		"webgl_loader_workertaskmanager",
```

Comment:
No longer in alphabetical order 🤓

Reply:
It was late. 😉

Reply:
Corrected, rebased and ast three commits are squashed